kpi-tracker/src/main.rs

```diff
@@ -209,48 +256,58 @@ fn get_account_transaction_details(
 fn to_block_events(block_hash: BlockHash, block_item: BlockItemSummary) -> Vec<BlockEvent> {
     let mut events: Vec<BlockEvent> = Vec::new();
 
-    match block_item.details {
+    match &block_item.details {
         AccountTransaction(atd) => {
-            let details = get_account_transaction_details(&atd, block_hash);
-            println!(
-                "TRANSACTION:\nhash: {}\ndetails: {:?}",
-                block_item.hash, &details
-            ); // Logger debug
-            let event = BlockEvent::AccountTransaction(block_item.hash, details);
+            log::debug!("TRANSACTION: {}", block_item.hash);
```

Comment:
I recommend doing proper logs. Not all-caps :)

Reply:
I tried with better descriptions and added the transaction relations found in there as well for each transaction.